eng/common/pipelines/templates/jobs/spec-gen-sdk.yml

```diff
@@ -0,0 +1,210 @@
+parameters:
+  - name: SpecRepoCommit
+    type: string
+    default: 'HEAD'
+  - name: SdkRepoCommit
+    type: string
+    default: 'HEAD'
+  - name: ConfigType
+    type: string
+    values:
+      - 'TypeSpec'
+      - 'OpenAPI'
+    default: 'TypeSpec'
+    displayName: 'Configuration Type'
+  - name: ConfigPath
+    type: string
+    default: 'specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml'
+    displayName: 'Path to TypeSpec or OpenAPI Configuration file'
+  - name: ApiVersion
```

Comment:
Could you delete `ApiVersion` for now and we will add it when all the downstream tools support it?

Reply:
I was talking with Ray about this yesterday and although some users will need this functionality, it is not the most common path and on the contrary, it can create confusion if users see it available.
If a user needs such a specific flow, they can generate the SDKs following the official documentation.

I want to make sure we get data of the number of users that run into this specific case. With that data we can re-evaluate the exposure of this functionality